Sources/Sentry/Public/SentryOptions.h

```diff
@@ -271,6 +271,23 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableSwizzling;
 
+/**
+ * If ANR (Application Not Responding) should be enabled or not. For now, this feature only prevents
+ * ANRs from being falsely detected as out of memory errors. The SDK doesn't report actual ANRs yet.
+ * Default is YES.
+ */
+@property (nonatomic, assign) BOOL enableANRTracking;
```

Comment:
I think this can be confusing. Default is `YES` but no events will be captured. So one day we'll make a release where we'll start sending events, without changing the default value from false to true.

To avoid reporting ANR as OOM we shouldn't need to rely on a user opt-in in to ANR, right? So if that's the case, we could default this to `false` (or do we need to have this at all?)

Otherwise we need a huge disclaimer: Opt-in out of this feature will cause misleading OOM errors to be reported. Which sounds like a side effect we don't want to have

Reply:
I believe ANR tracking should be enable as part of OOM tracking process. 
We should add this flag later when we are ready to send ANR report to sentry, but only disable the ANR if both this flag and OOM tracking are disabled.

Reply:
Yes, it makes sense. I should have removed this after deciding not to do an ANR feature.